### DISCUSSION TITLE
New Discussions section/category for announcements such as release?

### DISCUSSION THREAD

**Discussion by zhenhuaw-me** [onnx/onnx#5217] [Q&A]
We announced release in Discussions, e.g. https://github.com/onnx/onnx/discussions/5203. But that is a Q-A template.

Maybe we can create a new section/category for announcements such as release. Such section/category requires maintainer permission. @onnx/sig-archinfra-approvers Does this make sense to you?

<img width="867" alt="截屏2023-05-10 下午9 03 48" src="https://github.com/onnx/onnx/assets/4936589/bfbd18de-0a46-42c9-adae-b5ab6a9c1cf3">

**Comment by jcwchen** [comment]
Hi @zhenhuaw-me,
Good idea! I have added a category names "Release" and set previous discussions for 1.14.0, 1.13.1, 1.13.0 as category "Release".

**Reply by zhenhuaw-me** [reply]
Awesome!
